src/jit/compiler.cpp

```diff
@@ -7202,32 +7202,30 @@ void JitTimer::PrintCsvHeader()
 
     CritSecHolder csvLock(s_csvLock);
 
-    FILE* fp = _wfopen(jitTimeLogCsv, W("r"));
-    if (fp == nullptr)
+    if (FILE* fp = _wfopen(jitTimeLogCsv, W("a")))
```

Comment:
I just want to restict the scope of ''fp'' in the if block. I'll revise the code style as you suggested.